### PR TITLE
chore(storage): change reference link to cloud rad

### PR DESCRIPTION
**Pull request by danielduhh**
Update the storage README to point to Cloud RAD GO storage reference docs: https://cloud.google.com/go/docs/reference/cloud.google.com/go/storage/latest.

### CODE REVIEW

File: storage/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-## Cloud Storage [![Go Reference](https://pkg.go.dev/badge/cloud.google.com/go/storage.svg)](https://pkg.go.dev/cloud.google.com/go/storage)
+## Cloud Storage [![Go Reference](https://pkg.go.dev/badge/cloud.google.com/go/storage.svg)]([https://pkg.go.dev/cloud.google.com/go/storage](https://cloud.google.com/go/docs/reference/cloud.google.com/go/storage/latest))
```
Comment (danielduhh) [inline storage/README.md:1]: Not sure what to do with this badge....

Comment (tbpg) [reply storage/README.md:1]: I feel like the badge should stick with pkg.go.dev, since it's standard across Go packages.

**Comment by tbpg** [comment]
Thank you!

File: storage/README.md (hunk 1 above)
Comment (danielduhh) [reply storage/README.md]: Agreed. Switched it back

**Review by codyoss — APPROVED** [review]

**Review by tbpg — APPROVED** [review]
Thank you!

**Pull request merged**